cmd/wego/main.go

```diff
@@ -33,6 +35,15 @@ func configureLogger() {
 func main() {
 	rootCmd.PersistentFlags().BoolVarP(&options.verbose, "verbose", "v", false, "Enable verbose output")
 	rootCmd.AddCommand(version.Cmd)
+
+	if checkResponse, err := checkpoint.Check(&checkpoint.CheckParams{
```

Comment:
@palemtnrider @jrryjcksn  Would it be a good idea to use github api to check latest releases instead? In order to `go-checkpoint` to work we need to manually add a record in the DB throught a web page which is easily forgotten and easier to make a mistake in any of the forms fields. Mark and I looked at the web page a few days ago and the DB was outdated, the last release for wksctl  was `0.1.0-alpha.1`. Not sure if the info generated by this is still helpfull. I think if we can have less manuall steps like this it would be better. Or if we can create that record automatically after making a release, it would be cool too. 🙂

Reply:
Makes sense i can look into it

Reply:
So turns out i never looked too much at version.go That file already does the check by calling the releases endpoint.